socialDistribution/app/ajax_views.py

```diff
@@ -189,6 +189,18 @@ def get_node_host(nickname):
         return None
     return foreignNode.base_url
 
+@require_http_methods(["GET"])
+def get_foreign_nodes(request):
+    foreignNodes = ForeignAPINodes.objects.all()
+    data = []
+    for node in foreignNodes:
+        data.append({
+            'base_url': node.base_url,
+            'token': node.getToken()
+        })
+
+    return JsonResponse({'nodes': json.dumps(data)})
```

Comment:
Its not I going to use then I decided not to since I wanted to filter authors in the other endpoints on the server side (easy database stuff like following and such)